release/cmd/hashrelease.go

```diff
@@ -231,6 +233,13 @@ func hashreleaseSubCommands(cfg *Config) []*cli.Command {
 				if reg := c.StringSlice(registryFlag.Name); len(reg) > 0 {
 					opts = append(opts, calico.WithImageRegistries(reg))
 				}
+				if !c.Bool(publishHashreleaseImageFlag.Name) {
```

Comment:
do we really want to execute this if publishHashreleaseImageFlag is NOT true? or is the `!` a typo?

Reply:
`publishHashreleaseImageFlag` is for publishing images built locally by `hashrelease build`.
We need the list of components when the images have already been published else where